src/utils/axios.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-param-reassign */
+/**
+ * An axios proxy to attach the base url and authentication token
+ */
+import axios from 'axios';
+
+const axiosInstance = axios.create({
+  baseURL: process.env.API_ROUTE
+});
+
+axiosInstance.interceptors.request.use((config) => {
+  config.headers.Authorization = `BEARER ${localStorage.getItem('token')}`;
```

Comment:
Could you also add a way for axios to automatically reject error responses?  You can intercept the response to do this

Reply:
Please rename this to http i.e create a module for this i.e. http/index.js and also add the test from utils.spec.js to http/http.spec.js. Thanks

Reply:
should this still be in the utils folder?

Reply:
axios automatically throws an unsucessfull request as an error

Reply:
Really? So how come Gilbert's catch block isn't catching 400s and 500s?

Reply:
He wrapped it in a try/catch

Reply:
Isn't that why it should catch the errors?

Reply:
He wrapped the actual Axios request and the saga.